### PR TITLE
SHARE-166 Followers and other whom I follow not all shown

### PR DESCRIPTION
**Pull request by maxfazeli-dev**
---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [X] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [X] Choose the proper base branch (*develop*)
- [X] Confirm that the changes follow the project’s coding guidelines
- [X] Verify that the changes generate no warnings and errors 
- [X] Verify to commit no other files than the intentionally changed ones
- [X] Include reasonable and readable tests verifying the added or changed behavior
- [X] Verify that all tests are passing, if not please state the test cases in the [section](#Tests) below
- [X] Perform a self-review of the changes
- [X] Stick to the project’s git workflow (rebase and squash your commits)
- [X] Verify that your changes do not have any conflicts with the base branch
- [X] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [X] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

### CODE REVIEW

File: tests/behat/features/bootstrap/WebFeatureContext.php [hunk 1]
```diff
@@ -3144,6 +3144,55 @@ public function thereAreProgramsWithALargeDescription(TableNode $table)
...
+     * @var $em           EntityManager
+     */
+    $user_manager = $this->kernel->getContainer()->get(UserManager::class);
+    $user = null;
+    $users = $table->getHash();
+    $followedUser = $user_manager->createUser();
+    $followedUser->setUsername($users[0]['name']);
+    $followedUser->setEmail($users[0]['name'].'@pocketcode.org');
+    $followedUser->setAdditionalEmail('');
+    $followedUser->setPlainPassword('123456');
```
Comment (dmetzner) [inline tests/behat/features/bootstrap/WebFeatureContext.php]: use self::DEFAULT_PASSWORD instead ;)

File: tests/behat/features/web/many_follower.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Feature: User has a large number of follower (> 100).
+
+  Background:
+    Given 501 users follow:
+      | id | name      |
+      | 0  | user0     |
+
+  Scenario: Follower notification number in side menu should be same as profile follower number
+    Given I log in as "user0"
+    And I am on "/pocketcode/user"
```
Comment (dmetzner) [inline tests/behat/features/web/many_follower.feature]: use _app_ instead of _pocketcode_

File: tests/behat/features/web/many_follower.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    And I wait for the page to be loaded
+    Then I should see "My Profile"
+    And I wait for the page to be loaded
+    And I should see "Follower (500)"
+    When I am on "/pocketcode/user/"
```
Comment (dmetzner) [inline tests/behat/features/web/many_follower.feature]: you are already on this page.. use just  When I open the menu

File: tests/behat/features/web/many_follower.feature [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    Given 501 users follow:
+      | id | name      |
+      | 0  | user0     |
+
+  Scenario: Follower notification number in side menu should be same as profile follower number
+    Given I log in as "user0"
+    And I am on "/pocketcode/user"
+    And I wait for the page to be loaded
+    Then I should see "My Profile"
+    And I wait for the page to be loaded
```
Comment (dmetzner) [inline tests/behat/features/web/many_follower.feature]: unnecessary wait, you are already on the page and have already waited.

**Review by dmetzner — CHANGES_REQUESTED** [review]
Please just fix some minor stuff and it can be approved.

- Our public routes should always use **app** and not a specific flavor.

**Review by dmetzner — APPROVED** [review]

**Comment by HCrane** [comment]
@hungryapeman Please push the Ticket inside the Jira to Codereview!

**Pull request merged**